postreise/analyze/transmission/utilization.py

```diff
@@ -9,9 +9,7 @@ def get_utilization(branch, pf, median=False):
     :param pandas.DataFrame branch: branch data frame.
     :param pandas.DataFrame pf: power flow data frame.
     :param boolean median: take medians of pf for utilization calculation
-    :return: (*pandas.DataFrame*) -- normalized power flow data frame. For a
-        given line and hour, value is one if power flowing on the line reaches
-        capacity of the line.
+    :return: (*pandas.DataFrame*) -- normalized power flow data frame.
```

Comment:
I think the context of how we normalized matters here. Or, we could do something like `power flow data frame (per-unit)`.

Reply:
Done